packages/engine/Source/Scene/Cesium3DTilesVoxelProvider.js

```diff
@@ -89,6 +89,9 @@ function Cesium3DTilesVoxelProvider(options) {
   /** @inheritdoc */
   this.maximumTileCount = undefined;
 
+  /** @inheritdoc */
+  this.availableLevels = undefined;
```

Comment:
To match the VoxelProvider interface his needs to be defined as a getter, right?

Reply:
Yes, same for all 14 of these properties. I was trying to do a minimal fix and leave the refactor for https://github.com/CesiumGS/cesium/pull/12432

Reply:
Fair, but let's just do this one correctly to minimize conflicts.